templates/CRM/common/civicrm.settings.php.template

```diff
@@ -308,15 +307,15 @@ if (!defined('CIVICRM_DOMAIN_ID')) {
   define( 'CIVICRM_DOMAIN_ID', 1);
 }
 
-/** 
- * Setting to define the environment in which this CiviCRM instance is running. 
- * Note the setting here must be value from the option group 'Environment', 
- * (see Administration > System Settings > Option Groups, Options beside Environment) 
- * which by default has three option values: 'Production', 'Staging', 'Development'. 
+/**
+ * Setting to define the environment in which this CiviCRM instance is running.
+ * Note the setting here must be value from the option group 'Environment',
+ * (see Administration > System Settings > Option Groups, Options beside Environment)
+ * which by default has three option values: 'Production', 'Staging', 'Development'.
  * NB: defining a value for environment here prevents it from being set
- * via the browser. 
- */ 
-// $civicrm_setting[CRM_Core_BAO_Setting::DEVELOPER_PREFERENCES_NAME]['environment'] = 'Production'; 
+ * via the browser.
+ */
+// $civicrm_setting['domain']['environment'] = 'Production';
```

Comment:
@totten isn't it 'Developer'? Not domain?

Reply:
If you want the verbose/legacy style, then you'd need to lookup the exact group name. In this case, it would be `Developer Preferences`.

In 4.7.0+, all those things are just aliases for `domain`: https://docs.civicrm.org/dev/en/master/api/changes/#470-global-civicrm_setting-multiple-changes